lib/active_record/connection_adapters/spanner/schema_definitions.rb

```diff
@@ -26,7 +26,7 @@ def references *args, **options
 
       class Table < ActiveRecord::ConnectionAdapters::Table
         def primary_key name, type = :primary_key, **options
-          type = :string
+          type = :string # rubocop:disable Lint/ShadowedArgument
```

Comment:
This _reassignment_ code says `we use :string not :primary_key` explicitly. (but ,cop warning)
We can also write `super name, :string, **options`.

I think either is fine.